routing/routes.go

```diff
@@ -1433,6 +1433,28 @@ func NewRoutes() (r *Routes) {
 			handlers.Iter8ExperimentGetYaml,
 			true,
 		},
+
+		// swagger:route POST /stats/metrics stats metricsStats
+		// ---
+		// Produces metrics statistics
+		//
+		// 		Produces:
+		//		- application/json
+		//
+		//		Schemes: http, https
+		//
+		// responses:
+		//    400: badRequestError
+		//    503: serviceUnavailableError
+		//		500: internalError
+		//		200: metricsStatsResponse
+		{
+			Name:          "MetricsStats",
+			Method:        "POST",
+			Pattern:       "/api/stats/metrics",
+			HandlerFunc:   handlers.MetricsStats,
+			Authenticated: true,
+		},
```

Comment:
Note: we now have a molecule test "api-test" that can do basic tests again our API endpoints. If you would like to add a test for this new "api/stats/metrics" that runs nightly, you can add a `stats-metrics.test.yaml` [here](https://github.com/kiali/kiali-operator/tree/master/molecule/api-test). You can look at the other *.test.yaml files in there for examples - we do have a very basic `metrics.test.yaml` already - you can see it [here](https://github.com/kiali/kiali-operator/blob/master/molecule/api-test/metrics.test.yaml).

Reply:
thanks, I will take a look